htdocs/core/lib/functions.lib.php

```diff
@@ -222,7 +222,7 @@ function getDolUserInt($key, $default = 0, $tmpuser = null)
 		'fichinter' => 'intervention',
 		'product_fournisseur_price' => 'productsupplierprice',
 		'product_price' => 'productprice',
-		'projet'  => 'project',
+		'project'  => 'projet',
```

Comment:
The key is the old module name!

Reply:
This looks like the better way to do "it" - maybe refactor this to a common function :
https://github.com/Dolibarr/dolibarr/blob/55e201f63fb3bcaeced946bf8e4f3f59d004ce51/htdocs/core/lib/admin.lib.php#L1122-L1134